cmake/modules/ROOTMacros.cmake

```diff
@@ -210,7 +210,7 @@ MACRO (GENERATE_ROOT_TEST_SCRIPT SCRIPT_FULL_NAME)
                   )
   ENDIF(FAIRROOTPATH)
 
-  EXEC_PROGRAM(/bin/chmod ARGS "u+x  ${new_path}/${shell_script_name}")
+  EXEC_PROGRAM(/bin/chmod ARGS "u+x  ${new_path}/${shell_script_name}" OUTPUT_VARIABLE tmp)
```

Comment:
Instead of exec_program which is deprecated use execute_process which also allows to suppress the output to stdout

execute_process(COMMAND /bin/chmod u+x ${new_path}/${shell_script_name}
                             OUTPUT_QUIET)

Reply:
Ok, will do